Client/game_sa/CHandlingEntrySA.cpp

```diff
@@ -19,47 +19,54 @@ extern CGameSA* pGame;
 CHandlingEntrySA::CHandlingEntrySA()
 {
     // Create a new interface and zero it
-    m_pHandlingSA = new tHandlingDataSA;
-    memset(m_pHandlingSA, 0, sizeof(tHandlingDataSA));
-    m_bDeleteInterface = true;
+    if (m_HandlingSA = std::make_unique<tHandlingDataSA>())
+    {
+        MemSet(m_HandlingSA.get(), 0, sizeof(tHandlingDataSA));
+    }
 }
 
-CHandlingEntrySA::CHandlingEntrySA(tHandlingDataSA* pOriginal)
+CHandlingEntrySA::CHandlingEntrySA(const tHandlingDataSA* const pOriginal)
 {
     // Store gta's pointer
-    m_pHandlingSA = nullptr;
-    m_bDeleteInterface = false;
-    memcpy(&m_Handling, pOriginal, sizeof(tHandlingDataSA));
-}
-
-CHandlingEntrySA::~CHandlingEntrySA()
-{
-    if (m_bDeleteInterface)
+    m_HandlingSA = nullptr;
+    if (pOriginal)
     {
-        SAFE_DELETE(m_pHandlingSA);
+        MemCpy(&m_Handling, pOriginal, sizeof(tHandlingDataSA));
     }
 }
 
 // Apply the handlingdata from another data
-void CHandlingEntrySA::Assign(const CHandlingEntry* pEntry)
+void CHandlingEntrySA::Assign(const CHandlingEntry* const pEntry) noexcept
 {
     if (!pEntry)
         return;
 
-    // Copy the data
-    const CHandlingEntrySA* pEntrySA = static_cast<const CHandlingEntrySA*>(pEntry);
-    m_Handling = pEntrySA->m_Handling;
+    try
+    {
+        // Copy the data
+        const CHandlingEntrySA* const pEntrySA = static_cast<const CHandlingEntrySA const*>(pEntry);
+        m_Handling = pEntrySA->m_Handling;
+    }
+    catch (...)
+    {
+    }
```

Comment:
Why do you need all these try-catch blocks?

Reply:
@G-Moris Don't mark this as resolved before clarifying.

Reply:
In what sense should I understand this question?

Reply:
Added try catch blocks are useless and don't catch any exceptions.
C++ doesn't generate exceptions for invalid memory access. It looks like you are expecting that.

Reply:
Has it been fixed after the new PR? If not, please provide examples of your comments besides the current one.